client/src/components/EventCard.tsx

```diff
@@ -47,14 +49,56 @@ export const EventCard: React.FC<EventCardProps> = ({ event }) => {
       )}
     </>
   );
-  const eventStatus = event.canceled
-    ? 'Canceled'
-    : new Date(event.start_at) < new Date()
-    ? 'Passed'
-    : 'Upcoming';
-  const canceledStyle = { 'data-cy': 'event-canceled', color: 'red.500' };
+  enum EventStatus {
+    canceled = 'Canceled',
+    running = 'Running',
+    ended = 'Ended at',
+    upcoming = 'Upcoming',
+  }
+
+  const statusToStyle = {
+    [EventStatus.canceled]: { 'data-cy': 'event-canceled', color: 'red.500' },
+    [EventStatus.running]: {
+      color: 'gray.00',
+      backgroundColor: 'gray.45',
+      paddingInline: '.3em',
+      borderRadius: 'sm',
+    },
+    [EventStatus.ended]: { color: 'gray.45', fontWeight: '400' },
+    [EventStatus.upcoming]: {},
+  };
+
+  const canceled = event.canceled;
+  const isStartDateInPast = isPast(new Date(event.start_at));
+  const isEndDateInFuture = isFuture(new Date(event.ends_at));
+  const getEventStatus = ({
+    canceled,
+    isStartDateInPast,
+    isEndDateInFuture,
+  }: {
+    canceled: boolean;
+    isStartDateInPast: boolean;
+    isEndDateInFuture: boolean;
+  }) => {
+    if (canceled) return EventStatus.canceled;
+    if (!isStartDateInPast) return EventStatus.upcoming;
+    if (isEndDateInFuture) return EventStatus.running;
+    return EventStatus.ended;
+  };
+  const eventStatus = getEventStatus({
+    canceled,
+    isStartDateInPast,
+    isEndDateInFuture,
+  });
```

Comment:
```suggestion
  const hasEnded = isPast(new Date(event.ends_at));
  const getEventStatus = ({
    canceled,
    hasStarted,
    hasEnded,
  }: {
    canceled: boolean;
    hasStarted: boolean;
    hasEnded: boolean;
  }) => {
    if (canceled) return EventStatus.canceled;
    if (hasEnded) return EventStatus.ended;
    if (hasStarted) return EventStatus.running;
    return EventStatus.upcoming;
  };
  const eventStatus = getEventStatus({
    canceled: event.canceled,
    hasStarted: isPast(new Date(event.start_at)),
    hasEnded,
  });
```
This is a little bikesheddy, but I think this is a touch nicer.  We avoid the negation by re-ordering the ifs and `hasStarted` and `hasEnded` are more concise while still being technically accurate.